docs/geedocs/docsrc/answer/geeServerAdmin/publishDefaultDatabase.rst

```diff
@@ -0,0 +1,33 @@
+|Google logo|
+
+========================
+Publish default database
+========================
+
+.. container::
+
+   .. container:: content
+
+      Google Earth Enterprise (GEE) Server allows users to publish one 3D
+      database as the default database. This is the database that will be
+      served when Earth Client signs into the base URL without including
+      a database name.
+
+      .. rubric:: Publishing as Default
+
+      To make a 3D database the default database, simply check the "Default
```

Comment:
We should also mention how to set the default database from the command line here. I think that is already in the docs somewhere, but this is a more logical place for it.

Reply:
Added a note and example for the command line option.